packages/postgres-database/src/simcore_postgres_database/utils_projects_nodes.py

```diff
@@ -40,10 +40,25 @@ class ProjectNodesDuplicateNodeError(BaseProjectNodesError):
 class ProjectNodeCreate(BaseModel):
     node_id: uuid.UUID
     required_resources: dict[str, Any] = Field(default_factory=dict)
+    key: str
+    version: str
+    label: str
+    progress: float | None = None
+    thumbnail: str | None = None
+    input_access: dict[str, Any] | None = None
+    input_nodes: list[str] | None = None
+    inputs: dict[str, Any] | None = None
+    inputs_units: dict[str, Any] | None = None
+    output_nodes: list[str] | None = None
+    outputs: dict[str, Any] | None = None
+    run_hash: str | None = None
+    state: dict[str, Any] | None = None
+    parent: str | None = None
+    boot_options: dict[str, Any] | None = None
 
     @classmethod
     def get_field_names(cls, *, exclude: set[str]) -> set[str]:
-        return {name for name in cls.model_fields.keys() if name not in exclude}
+        return cls.model_fields.keys() - exclude
```

Comment:
interesting ... had to check to "believe". I did not know dict_keys could operate with sets this way :-)
```
>>> from pydantic import *
>>> class A(BaseModel):
...    x: int
... 
>>> A.model_fields.keys()
dict_keys(['x'])



>>> A.model_fields.keys() - {"foo"}
{'x'}
>>> A.model_fields.keys() - {"x"}
set()
>>> A.model_fields.keys() - {"bar"}
{'x'}
```
thx